vspace/src/main/java/edu/asu/diging/vspace/referenceExpose/ReferenceContext.java

```diff
@@ -0,0 +1,28 @@
+package edu.asu.diging.vspace.referenceExpose;
```

Comment:
i'm fairly certain i've made comments on the rest of the code before. (in the draft pr maybe?), although I can't find them. Please check/.

Reply:
Yes Julia. You made comments on vspc-147
https://github.com/diging/virtual-spaces-2.0/pull/228
I have worked on them earlier also and merged into this and reopened the PR. 
And for this now I have resolved all comments. 
Shall I go ahead and close the pull request link I mentioned above as we are tracking every thing in the current pull request!